layouts/partials/discourse.html

```diff
@@ -0,0 +1,13 @@
+<!-- layouts/partials/discourse.html -->
```

Comment:
Not really.. while we're debugging the comments it is nice to know the partial actually loaded.